dask_kubernetes/operator/daskcluster.py

```diff
@@ -111,3 +167,78 @@ async def daskcluster_create(spec, name, namespace, logger, **kwargs):
         f"A scheduler service has been created called {data['metadata']['name']} in {namespace} \
         with the following config: {data['spec']}"
     )
+
+    data = build_worker_group_spec("default", spec.get("image"), spec.get("workers"))
+    kopf.adopt(data)
+    api = kubernetes.client.CustomObjectsApi()
+    worker_pods = api.create_namespaced_custom_object(
+        group="kubernetes.dask.org",
+        version="v1",
+        plural="daskworkergroups",
+        namespace=namespace,
+        body=data,
+    )
+    logger.info(
+        f"A worker group has been created called {data['metadata']['name']} in {namespace} \
+        with the following config: {data['spec']}"
+    )
+
+
+@kopf.timer("daskworkergroup", interval=5.0)
```

Comment:
I am curious why this is on a timer?

Reply:
I think I added that because the function was not working without it. I'll look at this again and update you.